build.bat

```diff
@@ -0,0 +1,10 @@
+@echo off
```

Comment:
I suggest we use xtask (see https://github.com/massalabs/massa/pull/4005)
this way we could have only one version of build.xxx
what do you think ?

Reply:
Closed in favor of a more recent comment that supersede this one